logagg_collector/collector_command.py

```diff
@@ -0,0 +1,108 @@
+import os
+import socket
+
+from basescript import BaseScript
+from deeputil import AttrDict
+import tornado.ioloop
+import tornado.web
+from kwikapi.tornado import RequestHandler
+from kwikapi import API
+from logagg_utils import start_daemon_thread
+
+from .collector import LogCollector, CollectorService
+
+class LogaggCollectorCommand(BaseScript):
+    DESC = 'Logagg command line tool'
+
+    def parse_master_args(self, master_arg):
+        '''
+        Parse master arguments
+        '''
+        # Collector running with or without master
+        if not self.args.no_master:
```

Comment:
maybe we can use if self.args.no_master: return and then move the block inside if condition to its left by one indent. It is difficult to remember more conditions while going inside a `if` block.